src/ctia/store_service.clj

```diff
@@ -0,0 +1,22 @@
+(ns ctia.store-service
+  (:require [schema.core :as s]))
+
```

Comment:
Since protocol functions don't support varargs, the protocol counterparts for `ctia.stores/{read,write}-store` will not be able to support varargs. To avoid changing existing code, these function "lift" the 2 argument service functions to support varargs.